firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -28,6 +28,15 @@ def test_focus_and_clear(self):
         locationbar.clear()
         self.assertEqual(locationbar.value, '')
 
+    def test_gobutton(self):
+        # This replaces http://hg.mozilla.org/qa/mozmill-tests/file/default/firefox/tests/functional/testAwesomeBar/testGoButton.js
```

Comment:
This line is too long and travis fails because of it. Maybe do it like Bob here:
https://github.com/rbillings/firefox-ui-tests/blob/gobutton/firefox_ui_tests/functional/awesomebar/test_star_in_autocomplete.py#L11